matter/siwx917/BRD4325B/autogen/RTE_Components.h

```diff
@@ -0,0 +1,32 @@
+// This file is autogenerated by Silicon Labs SLC.
```

Comment:
Please specify which version of GSDK these files were taken from. Does the content of the rest of the files in this part of the repo correspond to that GSDK version? In other words, after this PR will the content match some version of GSDK?

Reply:
We tested with GSDK 4.3.0 it's working

Reply:
nvm3 contents are copied from gecko_sdk (4.2.0) and it's customized for 917 SoC in matter_support.
We are planning to remove the nvm3 from matter_support and use it from gecko_sdk. But, we need some changes from wiseconnect sdk 3.0 for this support (common and dual flash driver support for nvm3).
We will create new PR for matter_support after new modifications are tested with latest wiseconnect sdk 3.0 .
It will be good if this PR is approved for testing the SoC apps in studio.